doc/nrf/includes/nRF54H20_erase_UICR.txt

```diff
@@ -2,11 +2,16 @@
    Programming the nRF54H20 SoC can sometimes fail due to conflicts in the resource configuration.
    This can happen if, for example, an application programmed to the nRF54H20 SoC configured the UICRs for one or more cores in a way that is incompatible with the configuration required by the application you are trying to program on the SoC.
 
-   To fix this error and erase the UICRs, run the following commands::
+   To fix this error and erase the UICR for the application core, run the following command::
 
-      nrfutil device x-boot-mode-set --boot-mode safe --serial-number <serial_number>
-      nrfutil device erase --all --serial-number <serial_number> --core Application
-      nrfutil device erase --all --serial-number <serial_number> --core Network
-      nrfutil device x-boot-mode-set --boot-mode normal --serial-number <serial_number>
+      nrfutil device recover --core Application
```

Comment:
Can someone confirm this works?

Reply:
@shanthanordic  @ingeryp Could you confirm this?

Reply:
```
(py39-dev) inry@inry-linux:~/work/device_admin_adapter/$ nrfutil device recover --serial-number 001051164514 --core Application
✔️ Recovered 1051164514
(py39-dev) inry@inry-linux:~/work/device_admin_adapter/$ nrfutil device recover --serial-number 001051164514 --core Network
✔️ Recovered 1051164514
(py39-dev) inry@inry-linux:~/work/device_admin_adapter/$ nrfutil device --version
nrfutil-device 2.4.2 (f1dba09 2024-06-26)
commit-hash: f1dba096c99ddcc3562f27d8295f5d38683173b1
commit-date: 2024-06-26
host: x86_64-unknown-linux-gnu
build-timestamp: 2024-06-26T12:02:02.716754239Z
classification: nrf-external
```